src/stash/anchor.rs

```diff
@@ -153,7 +153,7 @@ impl Anchor {
             |mut data, (contract_id, node_id)| {
                 let id = Uint256::from_be_bytes(*contract_id.as_slice());
                 let vout = id % Uint256::from_u64(num_outs).unwrap();
-                let vout = vout.low_u64() as usize;
+                let vout = vout.low_u64() as usize + fee as usize;
```

Comment:
This assignment is not correct. We should get the modulus `num_outs` of the addition expression to make sure that `vout` does not exceed the number of transaction outputs.

```suggestion
                let vout = ((vout.low_u64() + fee as u64) % num_outs) as usize;
```